packages/openneuro-components/src/search-page/SearchSort.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Dropdown } from '../dropdown/Dropdown'
+import '../dropdown/dropdown.scss'
+import './search-sort.scss'
+
+export interface SearchSortProps {
+  items: [
+    {
+      label: string
+      value: string
+    },
+  ]
+  selected: {
+    label: string
+    value: string
+  }
+  setSelected: (selected: { label: string; value: string }) => void
+}
+
+export const SearchSort = ({
+  items,
+  selected,
+  setSelected,
+}: SearchSortProps) => {
+  return (
+    <Dropdown
+      label={
+        <div className="search-sort-list-label">
+          <b>SORT BY:</b> {selected.label}
+          <i className="fas fa-exchange-alt fa-rotate-90" />
+        </div>
+      }
+      children={
+        <div className="search-sort-dropdown-list">
+          <ul>
```

Comment:
Well, I have kept them in the specific component because each map requires different markup. Checkout packages/openneuro-components/src/user/UserMenu.tsx for example